package.json

```diff
@@ -8,11 +8,7 @@
     "dist"
   ],
   "dependencies": {
-    "@storybook/preset-create-react-app": "2.1.1",
-    "@types/moment": "^2.13.0",
```

Comment:
`moment` provide native TypeScript types so we don't need this `@types` package.